File-Formats/DocIO/word-to-image.md

```diff
@@ -596,6 +596,7 @@ N> 6. Word to Image conversion has the same limitations and unsupported elements
 N> 7. Different styles of underlines and borders are known limitations in Word to Image conversion in ASP.NET Core, Xamarin, Blazor, WinUI, and .NET MAUI platforms.
 N> 8. In ASP.NET Core, Blazor, Xamarin, WinUI and .NET MAUI platforms, to convert Word document to images we recommend you to use Word to image [assemblies](https://help.syncfusion.com/file-formats/docio/assemblies-required#converting-word-document-to-image) or [NuGet](https://help.syncfusion.com/file-formats/docio/nuget-packages-required#converting-word-document-to-image) as a reference in your application.
 N> 9. DocIO supports Word to image conversion in UWP application using DocIORenderer.
+N> 10. From v20.2, in addition to the previous NuGet packages, we recommend to use [SkiaSharp.NativeAssets.Linux v2.88.0-preview.209](https://www.nuget.org/packages/SkiaSharp.NativeAssets.Linux/2.88.0-preview.209) and [HarfBuzzSharp.NativeAssets.Linux v2.8.2-preview.209](https://www.nuget.org/packages/HarfBuzzSharp.NativeAssets.Linux/2.8.2-preview.209) NuGets to perform Word to Image conversion in Linux environment.
```

Comment:
Remove this wording "From v20.2,".
Start the sentence from "In addition to previous..."

Reply:
@MohanaselvamJothi -I have modified